Demos/Eyes/EyesWidget.cpp

```diff
@@ -106,17 +111,11 @@ Gfx::IntPoint EyesWidget::pupil_center(Gfx::IntRect& eyeball_bounds) const
     if (dx != 0 && abs(dx) >= abs(dy)) {
         double slope = dy / dx;
         double slope_squared = slope * slope;
-        max_distance_along_this_direction = 0.25 * sqrt(
-            (slope_squared + 1) /
-            (1 / width_squared + slope_squared / height_squared)
-        );
+        max_distance_along_this_direction = 0.25 * sqrt((slope_squared + 1) / (1 / width_squared + slope_squared / height_squared));
```

Comment:
@bugaevc should I revert this (and the one below) and ask clang-format to ignore them?

Reply:
Yes. Ideally, without leaving any special comments for it in the code. But if it's absolutely required, I guess we could live with the comments too.

Reply:
I'll try convincing clang-format to keep it as it was, if not, I'll add the on/off comments.